core/src/test/java/org/elasticsearch/search/sort/FieldSortBuilderTests.java

```diff
@@ -0,0 +1,18 @@
+package org.elasticsearch.search.sort;
```

Comment:
@juazugas just realized our fabolous CI testing robot doesn't work on the 2.4 branch any more, I just kicked of a maven build by hand and realized this file needs out standard licencing header just like the other tests in this folder. Can you simply add it to the top of the file so it passes the licence header checks?

Reply:
Ok.